dashboard/app/helpers/district_dropdown_constants.rb

```diff
@@ -0,0 +1,8 @@
+module DistrictDropdownConstants
```

Comment:
A common module sounds easiest to implement.  

That said, soon a User will have the same set of fields, and they will possibly, but not necessarily, be filled out (and we aren't planning on turning on validation to start with).  

How would you like the enrollment to handle that case?  Pre-populate the district dropdown with whichever fields we have?  Or something else?